lib/publish.js

```diff
@@ -76,6 +76,7 @@ async function publishChartToRegistry(configPath, registry, name, version) {
                 );
 
                 const chartArchive = stdout.split(":")[1].trim();
+                registry = `oci://${registry}`
```

Comment:
now we break setups where the `registry` starts with `oci://`

Reply:
reverted to the original commit